docs/topics/native/native-memory-manager.md

```diff
@@ -54,6 +54,15 @@ build script:
 
 ## Memory consumption
 
+### Check for memory leaks
+
+To access the memory manager metrics, call `kotlin.native.internal.GC.lastGCInfo()`. It returns statistics for the last
+run of the garbage collector.
+
+The statistics can be useful for debugging memory leaks or checking if there are any leaks when running tests.
```

Comment:
I would suggest clarifying, that leacks can happen only because of storing some objects into globals.

Also I think having an example would be nice. For example, use something like this

```
import kotlin.native.internal.*
import kotlin.test.*

class A

val global = mutableListOf<A>()

@OptIn(kotlin.ExperimentalStdlibApi::class)
fun getUsage() : Long {
	GC.collect()
	return GC.lastGCInfo!!.memoryUsageAfter["heap"]!!.totalObjectsSizeBytes
}

fun run() {
	global.add(A())
	// test would fail, if remove next line
	global.clear()
}

@Test
fun test() {
	val before = getUsage()
	// separate function is used to be sure, that all temporaries are cleaned
	run()
	val after = getUsage()
	assertEquals(before, after)
}
```

Reply:
Great suggestion! made a small fix on the class name and the opt-in annotation